### PR TITLE
Update PHP & dependencies

### PR DESCRIPTION
**Pull request by Nispeon**
#201 Get everything up to date with PHP 8 and Symfony 5.3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -52,8 +52,7 @@
         "symfony/phpunit-bridge": "*",
         "symfony/stopwatch": "*",
         "symfony/var-dumper": "^5.1",
```
Comment (lyrixx) [inline composer.json:54]: I would be better to use `^5.3` everywhere

File: config/bundles.php [hunk 2]
```diff
@@ -1,20 +1,10 @@
...
 
-/*
- * This file is part of the Secret Santa project.
- *
- * (c) JoliCode <coucou@jolicode.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
```
Comment (lyrixx) [inline config/bundles.php:11]: Why?

File: src/Controller/SantaController.php [hunk 3]
```diff
@@ -88,7 +88,7 @@ public function participants(Request $request, string $application): Response
         $errors = [];
 
         if ($request->isMethod('POST')) {
-            $selectedUsers = $request->request->get('users', []);
+            $selectedUsers = $request->request->all('users', []);
```
Comment (lyrixx) [inline src/Controller/SantaController.php:91]: This does not work like this.  There is no default parameter anymore. it must be `$request->request->all()['users'] ?? []`

File: config/bundles.php (hunk 2 above)
Comment (Nispeon) [reply config/bundles.php]: The file was re-written when I removed web-server-bundle, I ran cs-fixer to add it back.

File: src/Controller/SantaController.php (hunk 3 above)
Comment (Nispeon) [reply src/Controller/SantaController.php]: Indeed, I removed the useless parameters.

**Review by pyrech — APPROVED** [review]

**Pull request merged**